core/trino-main/src/test/java/io/trino/operator/BenchmarkGroupByHash.java

```diff
@@ -77,7 +78,13 @@
     public Object groupByHashPreCompute(BenchmarkData data)
     {
         GroupByHash groupByHash = new MultiChannelGroupByHash(data.getTypes(), data.getChannels(), data.getHashChannel(), EXPECTED_SIZE, false, getJoinCompiler(), TYPE_OPERATOR_FACTORY, NOOP);
-        data.getPages().forEach(p -> groupByHash.getGroupIds(p).process());
```

Comment:
why this is changed?

Reply:
The benchmarks need to ensure that the input pages are fully processed, which wasn't guaranteed to happen before with only a single call to `WorkProcessor#process()`.